apps/website/src/pages/index/index.vue

```diff
@@ -10,13 +10,24 @@ const successMessage: Ref = ref<HTMLDivElement>()
 const invalidMessage: Ref = ref<HTMLDivElement>()
 
 const email = ref('')
+const botTrapInput = ref('')
+const timeArrived = ref(new Date().getTime())
+
 async function onSubmit() {
```

Comment:
Could you move all the bot logic (18-19 and 25-30) into `checkForBots` or some better name that you come up with?

Reply:
done.